src/components/panels/transitions.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<panel-wrapper :content-class="['panel-transitions']">
+		<template slot="name">Transitions</template>
+		<div class="transitions" v-if="allTransitions">
+			<button
+			  class="transition"
+				v-for="transition in allTransitions"
+				:key="transition.name"
+				:class="[transition.name === currentTransition ? 'is-active' : 'is-inactive']"
+				@click="switchTransitions(transition.name)"
+				v-text="transition.name" />
+			<hr/>
+			<button
+				class="duration"
+				v-for="duration in staticDurations"
+				:class="[duration.duration === currentDuration ? 'is-active' : 'is-inactive']"
+				:key="duration.duration"
+				@click="changeDuration(duration.duration)">{{ duration.duration }}ms</button>
+	  </div>
+		<div v-else>
+			Quick Transition list is empty! Should minimally have 'Cut' and 'Fade' ... Something is wrong :(
+		</div>
+
+	</panel-wrapper>
+
+</template>
+
+<script> //TODO: determine active transition and add is-active is-inactive logic (also something like that for duration)
+import {mapState, mapActions} from 'vuex'
+
+import panelMixin from '../mixins/panel'
+
+	export default {
+
+		data() {
+			const data = {
+				staticDurations: [ {duration: 100}, {duration: 300}, {duration: 600}, {duration: 1000}, {duration: 1500} ]
```

Comment:
Maybe just array of numbers (`[100, 300, 600, ...]`), that way could get rid all of the `duration.duration` above

(Also this could be made into static computed property since it never changes)

```js
computed: {
    staticDurations() {
        return [ ... ]
    }
}
```

Reply:
wasn't sure how to do that, coudn't get the v-for to enumerate correctly. so made it an object instead, with duration: xxx